### PR TITLE
Done mentor part to Database project

### PR DESCRIPTION
**Pull request by kuzmiich**
Create table script tasks
create procedure to delete task
create views to task & userTrack

### CODE REVIEW

File: DIMS-Core.Database/DIMS-Core.Database/DIMS-Core.Database.refactorlog [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <Property Name="NewName" Value="TaskId" />
+  </Operation>
+  <Operation Name="Rename Refactor" Key="16dc267e-039f-4442-8712-59559a68713d" ChangeDateTime="09/06/2021 09:13:54">
+    <Property Name="ElementName" Value="[dbo].[Task].[Deadli]" />
+    <Property Name="ElementType" Value="SqlSimpleColumn" />
+    <Property Name="ParentElementName" Value="[dbo].[Task]" />
+    <Property Name="ParentElementType" Value="SqlTable" />
+    <Property Name="NewName" Value="DeadlineDate" />
+  </Operation>
+</Operations>
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/DIMS-Core.Database.refactorlog:17]: remove this file. it's redundant

File: DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿CREATE PROCEDURE DeleteTask AS
+
+DECLARE @id int;
+SET @id = 1;
+
+Delete Task WHERE Task.TaskId = @id;
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql:6]: delete from

File: DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+EXEC DeleteTask @id;
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql:8]: why do you need this call?

File: DIMS-Core.Database/DIMS-Core.Database/Tables/Tasks.sql [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[Tasks]
+(
+	[TaskId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [Name] NVARCHAR(50) NOT NULL, 
+    [Description] NTEXT NOT NULL,
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Tables/Tasks.sql:5]: nvarchar

File: DIMS-Core.Database/DIMS-Core.Database/Views/vTask.sql [hunk 5]
```diff
@@ -0,0 +1,3 @@
+﻿CREATE VIEW [dbo].[vTask]
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Views/vTask.sql:1]: plural name

**Review by PushMeNow — CHANGES_REQUESTED** [review]

File: DIMS-Core.Database/DIMS-Core.Database/DIMS-Core.Database.refactorlog (hunk 1 above)
Comment (kuzmiich) [reply DIMS-Core.Database/DIMS-Core.Database/DIMS-Core.Database.refactorlog:17]: Deleted

File: DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql (hunk 2 above)
Comment (kuzmiich) [reply DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql:6]: Fixed

File: DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql (hunk 3 above)
Comment (kuzmiich) [reply DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql:8]: Deleted

File: DIMS-Core.Database/DIMS-Core.Database/Views/vTask.sql (hunk 5 above)
Comment (kuzmiich) [reply DIMS-Core.Database/DIMS-Core.Database/Views/vTask.sql:1]: Fixed

**Review by PushMeNow — APPROVED** [review]

**Pull request merged**